it/uk/gov/hmrc/incorporatedentityidentificationfrontend/assets/TestConstants.scala

```diff
@@ -52,7 +53,7 @@ object TestConstants {
   val testCharitableIncorporatedOrganisationJourneyConfig: JourneyConfig = createTestJourneyConfig(CharitableIncorporatedOrganisation)
 
   private def createTestJourneyConfig(entityType: BusinessEntity) =
-    JourneyConfig(testContinueUrl, PageConfig(None, testDeskProServiceId, testSignOutUrl), entityType)
+    JourneyConfig(testContinueUrl, PageConfig(None, testDeskProServiceId, testSignOutUrl), entityType, testBusinessVerificationCheck)
```

Comment:
I think it would be better if we passed the businessVerificationCheck into this method, and then in the methods above this that use this we could set it as true. Then this can be changed in the next ticket to true/false when needed by tests